zypp/repo/SUSEMediaVerifier.cc

```diff
@@ -55,10 +56,16 @@ bool SUSEMediaVerifier::isDesiredMedia(const media::MediaAccessRef &ref)
   std::ifstream str(media_file.asString().c_str());
   std::string vendor;
   std::string id;
-#warning check the stream status
+
+  DBG << "reading data from repository media: " << media_file << endl;
   getline(str, vendor);
   getline(str, id);
 
+  DBG << "cached vendor: " << _media_vendor << endl;
+  DBG << "repo vendor: " << vendor << endl;
+  DBG << "cached id: " << _media_id << endl;
+  DBG << "repo id: " << id << endl;
+
```

Comment:
Logging the values is a good idea, if the function is about to return `false`. The good case is IMO not interesting. (store return value in `bool ret` and log if `false`)